src/_guides/language/language-tour.md

```diff
@@ -3342,35 +3340,34 @@ You can do this using `extends`.
 
 <?code-excerpt "misc/lib/language_tour/generics/base_class.dart" replace="/extends SomeBaseClass(?=. \{)/[!$&!]/g"?>
 {% prettify dart %}
-// T must be SomeBaseClass or one of its descendants.
 class Foo<T [!extends SomeBaseClass!]> {
-  // ···
+  // Implementation goes here...
+  String toString() => "Instance of 'Foo<$T>'";
 }
 
-class Extender extends SomeBaseClass {
-  // ···
-}
+class Extender extends SomeBaseClass {...}
 {% endprettify %}
 
 It's OK to use `SomeBaseClass` or any of its subclasses as generic argument:
 
-<?code-excerpt "misc/lib/language_tour/generics/base_class.dart (SomeBaseClass-ok)" replace="/Foo.\w+./[!$&!]/g"?>
+<?code-excerpt "misc/test/language_tour/generics_test.dart (SomeBaseClass-ok)" replace="/Foo.\w+./[!$&!]/g"?>
 {% prettify dart %}
 var someBaseClassFoo = new [!Foo<SomeBaseClass>!]();
 var extenderFoo = new [!Foo<Extender>!]();
 {% endprettify %}
 
-It's also OK to specify no generic argument:
+When you don't provide a type argument, it defaults to the type bound:
```

Comment:
"type bound" threw me — we should either introduce that phrase at the top of this section or use words we've used before, such as "type you specified with `extends`"

Reply:
Actually, maybe we should use the original words:

It’s also OK to specify no generic argument:

Reply:
Ok, reverting.